fimsauthz-api/fims-authz.yaml

```diff
@@ -291,7 +290,23 @@ components:
             For this server, this is satisfied by the use of a UUID.
         scope:
           description: See securityDefinitions section in this spec.
+          type: array
+          items:
+            type: string
+            example: "example.scope.name.read"
+          maxItems: 1
+          minItems: 1
+        authorities:
```

Comment:
client's authorities is a claim as well. Please review the description

Reply:
Are these "roles" in our prior lingo?  Seems a little odd to be returning.  Like who does this benefit?  Seems like an unnecessary information leak.  May need to discuss.

Reply:
Hi Archit,  From my reading, the UFAA does not include 'authorities' in the Claims set, in fact it refers to Certificate Authorities.  
I'd guess this  'authoritiies' term originated as  a Spring Security Framework specific  and it landed in our old client-authen-database, however, we should not have it in the JWT Claims set.